### PR TITLE
Update architecture.md

### PR DESCRIPTION
**Pull request by elctrogamr1**
Minor grammatical fixes.

### CODE REVIEW

**Comment by scanum** [comment]
[@JohanFreelancer9](https://github.com/JohanFreelancer9) Copy editing is needed for this PR. Thanks.

File: Exchange/ExchangeServer/architecture/architecture.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ manager: serdars
 
 # Exchange architecture
 
-Exchange use a single building block architecture that provides email services for deployments at all sizes, from small organizations to the largest multi-national corporations. This architecture is describe in the following diagram.
+Exchange uses a single building block architecture that provides email services for deployments at all sizes, from small organizations to the largest multi-national corporations. This architecture is described in the following diagram.
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeServer/architecture/architecture.md:21]:
```suggestion
Exchange uses a single building block architecture that provides email services for deployments of all sizes, from small organizations to the largest multi-national corporations. This architecture is described in the following diagram.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @elctrogamr1, kindly commit the additional suggested change. Many thanks.

**Comment by elctrogamr1** [comment]
Hi @JohanFreelancer9, I committed the additional change. By the way, this is my first PR ever, so please let me know if I missed anything. Thank you for the suggested change, too!

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@tiburd PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**